Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM docker.io/library/ruby:2
+
+# Copy and run the setup.
+COPY setup.sh /usr/local/bin/setup.sh
+RUN DEBIAN_FRONTEND=noninteractive apt update \
+    && apt install -y sudo \
+    && DEBIAN_FRONTEND=noninteractive SKIP_BUNDLE=1 /usr/local/bin/setup.sh \
+    && rm -rf /var/lib/apt/lists/*
+
+# Please mount the site in this directory.
+RUN mkdir -p /srv/site
+WORKDIR /srv/site
+VOLUME /srv/site
+
+# HTTP port.
+EXPOSE 4000
+
+# Live-reload port.
+EXPOSE 35729
+
+# All parameters passed to the container get routed through a script we create here.
+RUN echo "#!/bin/sh" >/usr/local/bin/init.sh \
+    && echo "set -e" >>/usr/local/bin/init.sh \
+    && echo "bundle config --local path 'vendor/bundle'" >>/usr/local/bin/init.sh \
```

Comment:
bundle config/install are already done in `setup.sh`
(and btw we do not need to repeat them in `run-server.sh` either; I often forget about this script and run the exec manually and that's really not needed)

Reply:
I made it conditional so that we can do it runtime rather than build time.